### PR TITLE
Test get history

### PR DESCRIPTION
**Pull request by cpennington**
@alawibaba @utkjad @doctoryes @macdiesel: Review?

### CODE REVIEW

**Comment by macdiesel** [comment]
:+1:

File: edx_user_state_client/tests.py [hunk 1]
```diff
@@ -268,58 +295,162 @@ def test_delete_many_last_field(self):
...
+        history = list(self.get_history(0, 0))
+
+        self.assertEquals(
+            [entry.state for entry in history],
+            [{'a': val} for val in reversed(xrange(10))]
+        )
+
+        self.assertEquals(
+            [entry.updated for entry in history],
+            sorted((entry.updated for entry in history), reverse=True)
```
Comment (doctoryes) [inline edx_user_state_client/tests.py]: How about a comment here to help a test reader out?

File: edx_user_state_client/tests.py [hunk 2]
```diff
@@ -268,58 +295,162 @@ def test_delete_many_last_field(self):
         self.assertItemsEqual(self.get_many(0, [0, 1]), [])
 
 
+class _UserStateClientTestHistory(_UserStateClientTestUtils):
+    """
+    Blackbox tests of basic XBlockUserStateClient history functionality.
+    """
```
Comment (doctoryes) [inline edx_user_state_client/tests.py]: There's no test below for `set_many` with `get_history`.

**Comment by doctoryes** [comment]
Initial review complete.

File: edx_user_state_client/tests.py (hunk 2 above)
Comment (cpennington) [inline edx_user_state_client/tests.py]: I added a test using `set_many`.

**Comment by doctoryes** [comment]
:+1:

**Pull request merged**